### DISCUSSION TITLE
How to specify accounts to the CLI

### DISCUSSION THREAD

**Discussion by dmshimself** [firefly-iii/firefly-iii#9120] [Firefly III]
I've edited a number of rules and want to run all of them against all of my accounts, but I'm having trouble knowing how to specify the account names.  I have several asset accounts, but to keep it simple I have an asset account called Visa and used

php artisan firefly-iii:apply-rules --accounts=Visa --all_rules --token=xxxxwhatever

and then get a message saying 

  [x]  Please make sure all accounts in --accounts are asset accounts or liabilities

Any advice appreciated

**Comment by JC5** [comment]
Great question! To make sure Firefly III recognizes the account, make sure you use the ID, not the name.

**Comment by dmshimself** [comment]
Great, but I cannot seem to find the ID.  I don't think its the account number or credit card number as I tried those and I cannot see anything else displayed on the account details which looks to be an ID.  Is there some clever CLI that can be used to show this detail?

**Reply by JC5** [reply]
The ID is visible in de URL when you visit `/accounts/show/x`, where X is the ID. You can also hover over the link from `/accounts/asset`.
